testing/btest/supervisor/config-scripts.zeek

```diff
@@ -36,7 +38,15 @@ event zeek_init()
 		{
 		Broker::peer("127.0.0.1", to_port(getenv("BROKER_PORT")));
 		node_output_file = open("node.out");
-		print node_output_file, "supervised node zeek_init()";
+
+@ifdef ( Pre::counter )
+		# Even though this else-block runs only in the supervised node,
+		# it still needs to parse correctly in the Supervisor, so we
+		# need to condition this with availability of the counter (which
+		# only exists in the supervised node).
+		++Pre::counter;
+		print node_output_file, fmt("supervised node zeek_init(), counter at %s", Pre::counter);
+@endif
```

Comment:
Is this block related to this PR or something else?

Reply:
This PR — the test verifies the load order by adding the counter in the post-base scripts, using it in the test script itself, and again in the post-user ones.